### PR TITLE
MLPAB-2126 Re-order attorney sign handlers to work when failure

### PR DESCRIPTION
**Pull request by hawx**

### CODE REVIEW

File: internal/page/attorney/sign_test.go [hunk 1]
```diff
@@ -305,6 +337,7 @@ func TestPostSign(t *testing.T) {
 		appData         page.AppData
 		form            url.Values
 		lpa             *lpastore.Lpa
+		sendsToLpaStore bool
```
Comment (acsauk) [inline internal/page/attorney/sign_test.go:340]: This looks like its unused at the monent.

**Review by acsauk — COMMENTED** [review]
Splitting the sign handler by attorney type is clearer now 👍 

One leftover testcase prop that probably needs to be removed then good to go.

File: internal/page/attorney/sign_test.go (hunk 1 above)
Comment (hawx) [reply internal/page/attorney/sign_test.go]: :rage1: I thought I'd removed that

**Review by acsauk — APPROVED** [review]
:shipit:

**Pull request merged**